enclave_apps/oak_functions_enclave_app/src/main.rs

```diff
@@ -23,7 +23,7 @@ extern crate alloc;
 use alloc::{boxed::Box, sync::Arc};
 
 use oak_core::samplestore::StaticSampleStore;
```

Comment:
Worth thinking if we also want to proxy this through the SDK, as otherwise enclave apps would need to continue depending on oak_core. 

Thoughts, @tiziano88?

Reply:
Sounds like a good idea to me.

Reply:
Also like this